code/modules/urist/modules/weather/area_climates.dm

```diff
@@ -0,0 +1,12 @@
+//'Climates' (definitions of weather probabilities per area as lists) go here. You can use weights as for pick().
+//NOTE: /clear weather's effect is overriden by all other types! If you have rain and clear, it's effectively rain.
+
+/area
+	var/list/climate = list(/obj/weathertype/clear) //all potential area weather types
+	var/list/weather = list(/obj/weathertype/clear) //current pick for all objects in area
+
+/area/jungle
```

Comment:
You can, it uses a decorated pick(), I had them but removed them for testing. I didn't check if for() respects them as one list object.